manifests/generated/monitoring/prom.yaml

```diff
@@ -158,6 +158,9 @@ data:
       - source_labels: [__meta_kubernetes_namespace]
         target_label: namespace
         action: replace
+    - job_name: pingapp 
```

Comment:
That will unfortunately not work. Couple of things:

1. Prometheus is in a separate container/pod. So localhost means same container. Since the pong app is running in other containers, we cannot access it on Prometheus container's 8080 port.
2. This file is generated from Go code. You can find source in `source/demo.mimic.go`. Changing it in YAML will be lost as soon as we change something in Go code and run `make gen`
3. I configured Prometheus already to automatically discover things. Particularly this job: https://github.com/AnaisUrlichs/observe-argo-rollout/blob/ae620469b4243939691b09578e9c3178ab53405c/source/demo.mimic.go#L451 is using [Kubernetes Discoverer](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config). The job's configuration is very flexible. I created it in such a way that as soon as you name port in your pod with `m-` prefix, Prometheus will immediately scrape it. 

Let me know if those makes sense! (:

Reply:
BUT.... Now I realized that wonder if that works with Rollouts Argo supplies. 🤔 You can leave that part to me, I can today get your ping app scraped by Prometheus (: